python/EVPFFT-GUI/evpfft_gui/EVPFFT_GUI.py

```diff
@@ -723,7 +722,10 @@ def write_input_file(BC_index):
             evpfft_lattice_input.write(modes)
             dimensions = str(int(self.INNumberOfVoxelsX.text())) + ' ' + str(int(self.INNumberOfVoxelsY.text())) + ' ' + str(int(self.INNumberOfVoxelsZ.text())) + '               x-dim, y-dim, z-dim\n'
             evpfft_lattice_input.write(dimensions)
-            dx, dy, dz = self.VoxelResolution
+#            dx, dy, dz = self.VoxelResolution
```

Comment:
Was this hard coding of 1 on purpose?  Does it get modified later?

Reply:
This is an old GUI. The new GUI does not hard code this in. The old version of the GUI (EVPFFT only) has been removed.

Reply:
Perfect.  Thanks!